src/MongoDB/Driver/ReadPreference.cpp

```diff
@@ -103,16 +103,16 @@ void HHVM_METHOD(MongoDBDriverReadPreference, _setReadPreferenceTags, const Arra
 	}
 }
 
-void HHVM_METHOD(MongoDBDriverReadPreference, _setMaxStalenessMS, int maxStalenessMS)
+void HHVM_METHOD(MongoDBDriverReadPreference, _setMaxStalenessSeconds, int maxStalenessSeconds)
 {
 	MongoDBDriverReadPreferenceData* data = Native::data<MongoDBDriverReadPreferenceData>(this_);
 
-	/* Validate that maxStalenessMS is not used with PRIMARY readPreference */
+	/* Validate that maxStalenessSeconds is not used with PRIMARY readPreference */
 	if (mongoc_read_prefs_get_mode(data->m_read_preference) == MONGOC_READ_PRIMARY) {
-		throw MongoDriver::Utils::throwInvalidArgumentException("maxStalenessMS may not be used with primary mode");
+		throw MongoDriver::Utils::throwInvalidArgumentException("maxStalenessSeconds may not be used with primary mode");
```

Comment:
The primary conflict should only be reported if `maxStalenessSeconds` is not equal to `NO_MAX_STALENESS`.